docs/concepts/tasks.md

```diff
@@ -32,7 +32,7 @@ Tasks are special because they receive metadata about upstream dependencies and
 
 Tasks also take advantage of automatic Prefect [logging](/concepts/logs/) to capture details about task runs such as runtime, tags, and final state.
 
-You can define your tasks within the same file as your flow definition, or you can define tasks within modules and import them for use in your flow definitions. All tasks must be called from within a flow. Tasks may not be called from other tasks.
+You can define your tasks within the same file as your flow definition, or you can define tasks within modules and import them for use in your flow definitions. Tasks may be called from other tasks, from within a flow, from within a subflow, or (as of `prefect 2.18.x`) from within another task.
```

Comment:
What is the difference between "tasks may be called from other tasks" and "from within another task"?

Reply:
ummmm.... nothing. lemme update

Reply:
updated in [rm duplicative language](https://github.com/PrefectHQ/prefect/pull/13154/commits/b27dd0c0a5a7f5e2787eb6dea3b3301debc498d3)